docs/antora/modules/configuration/pages/roll-cycle.adoc

```diff
@@ -153,7 +153,86 @@ SingleChronicleQueue queue = ChronicleQueue.singleBuilder("/timezone")
+ -m <min-archive-age>       Archive files older than this age. Can be specified as milliseconds since epoch or local datetime. Either this or -r must be provided.
+ -qbme <qbm-entries>        Maximum number of entries in QBM (if queue being archived is backing a QBM)
+ -qbmn <qbm-name>           QBM name (if queue is backing a QBM)
+ -qc <queue-config-file>    QueueReplicationConfig file containing queue configuration
+ -qn <queue-name>           Name of queue as used in queue-config-file
+ -r <roll-cycles-to-keep>   Number of roll cycles to keep when archiving. Either this or -m must be provided.
+ -vc <value-class>          Name of value class (if queue being archived is backing a QBM)
+----
+
+==== Command-Line Parameters
+
+|===
+| Parameter | Detail | Required?
+
+| -qc | Full or relative path to file to load configuration from for queue being archived. | Y
+| -qn | Name of queue being archived, as it appears in file specified by -qc. eg. `'queue1'` | Y
+| -aa | Archive action being performed. Accepted values are `ARCHIVE` (move roll files to archive directory) or `DELETE` (delete eligible roll files permanently) | Y
+| -m | Minimum age for files to be eligible for archiving, specified either in milliseconds since epoch, or as a datetime in the format `yyyy-MM-dd'T'HH:mm:ss` (eg. `2024-05-23T23:55:01`). | If `-r` is not specified
+| -r | Number of roll cycles to retain. If this option is combined with `-m`, the minimum number of roll cycle files will be retained before considering files for archiving. | If `-m` is not specified
+|===
+
+==== Queue-Backed-Map Support
+
+If the queue being archived is backing a `QueueBackedMap`, `ArchiveRollFiles` can provide additional utility by writing a full snapshot of the `QueueBackedMap` to the current roll cycle before archiving older roll cycle files, ensuring the `QueueBackedMap` contents are safely preserved.
+
+When archiving a queue backing a `QueueBackedMap` queue, the following parameters must be provided:
+
+|===
+| Parameter | Detail | Required?
+
+| -qbmn | Name of the `QueueBackedMap` | Y
+| -qbme | Maximum number of entries in the `QueueBackedMap` | Y
+| -kc | Fully qualified name of the `QueueBackedMap` key class | Y
+| -vc | Fully qualified name of the `QueueBackedMap` value class | Y
+
+|===
+
+==== Examples
+
+Invoking `ArchiveRollFiles` to delete all roll files older than the age specified for `queue1`.
+
+[source, shell script]
+----
+$ mvn exec:java -Dexec.mainClass="software.chronicle.enterprise.queue.util.ArchiveRollFilesMain"
+-Dexec.args="-qc config/queue-config.yaml -qn queue1 -m 2024-11-01T12:15:00 -aa DELETE"
+----
+
+Invoking `ArchiveRollFiles` to archive roll files older than the age specified for a `QueueBackedMap` queue.
+[source, shell script]
+----
+$  mvn exec:java -Dexec.mainClass="software.chronicle.enterprise.queue.util.ArchiveRollFilesMain"
+-Dexec.args="-qc /tmp/qbm-replication.yaml -qn queue1 -m 1707289226000 -aa ARCHIVE
```

Comment:
pls use nicely formatted date time

Reply:
Intent was to provide an example of `-m` with both conventions - so stick to formatted date time for both examples?

Reply:
I honestly can't see why anyone would use the milliseconds option. You have documented above that it is possible

Reply:
Understood - I've removed it from the documentation.